.github/workflows/e2e-functional.yml

```diff
@@ -89,38 +87,40 @@ jobs:
           wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.20.1/yq_linux_amd64 && chmod a+x /usr/local/bin/yq
           sudo apt-get update || true && sudo apt-get install -y ca-certificates libxtst-dev libpng++-dev gcc-aarch64-linux-gnu g++-aarch64-linux-gnu jq icnsutils graphicsmagick tzdata
           npm ci
+          npm install -g node-gyp
+          npm i robotjs
```

Comment:
Why are we installing this and `node-gyp` separately? Can we not just use what's already in the Desktop App? There is a script to do all of this already.